python/dgl/sampling/neighbor.py

```diff
@@ -8,10 +8,136 @@
 from .. import utils
 
 __all__ = [
+    'sample_etype_neighbors',
     'sample_neighbors',
     'sample_neighbors_biased',
     'select_topk']
 
+def sample_etype_neighbors(g, nodes, etype_field, fanout, edge_dir='in', prob=None,
+                           replace=False, copy_ndata=True, copy_edata=True, _dist_training=False):
```

Comment:
we don't need this argument. we can check the type of g to figure out if this is distributed training.